features-json/focusoptions-preventscroll.json

```
@@ -231,7 +231,7 @@
       "12":"n",
       "12.1":"n",
       "13":"n",
-      "13.1":"n",
+      "13.1":"y",
       "TP":"n"
```

```suggestion
      "TP":"y"
```
If it works in Safari 13.1, it probably also works in Technology Preview. I'd be curious to know if it works in Safari on IOS as well.